From: Hugh Dickins <hughd@google.com>
Subject: mm/swapfile: delete outdated pte_offset_map() comment
Date: Mon, 19 Jun 2023 23:50:00 -0700 (PDT)

Delete a triply out-of-date comment from add_swap_count_continuation():
1. vmalloc_to_page() changed from pte_offset_map() to pte_offset_kernel()
2. pte_offset_map() changed from using kmap_atomic() to kmap_local_page()
3. kmap_atomic() changed from using fixed FIX_KMAP addresses in 2.6.37.

Link: https://lkml.kernel.org/r/9022632b-ba9d-8cb0-c25-4be9786481b5@google.com
Signed-off-by: Hugh Dickins <hughd@google.com>
Signed-off-by: Andrew Morton <akpm@linux-foundation.org>
---

 mm/swapfile.c |    5 -----
 1 file changed, 5 deletions(-)

--- a/mm/swapfile.c~mm-swapfile-delete-outdated-pte_offset_map-comment
+++ a/mm/swapfile.c
@@ -3456,11 +3456,6 @@ int add_swap_count_continuation(swp_entr
 		goto out;
 	}
 
-	/*
-	 * We are fortunate that although vmalloc_to_page uses pte_offset_map,
-	 * no architecture is using highmem pages for kernel page tables: so it
-	 * will not corrupt the GFP_ATOMIC caller's atomic page table kmaps.
-	 */
 	head = vmalloc_to_page(si->swap_map + offset);
 	offset &= ~PAGE_MASK;
 
_
